lib/Marlin/Marlin/src/feature/prusa/crash_recovery.h

```diff
@@ -44,7 +44,7 @@ class Crash_s {
 
     struct crash_block_t {
         xyze_pos_t start_current_position; /// Starting logical position of the G-code
-        pos_t e_position;                  /// Starting physical position of the segment
+        float e_position;                  /// Starting physical position of the segment
```

Comment:
weird but OK

Reply:
pos_t was actually our addition. In 2.1.x there are aliases for integer counters, multiple axes, but for absolute logical position everything has just a float type for all axes so I went with it. No point in adding a type just for one exception.